src/components/ebay-alert-dialog/index.marko

```diff
@@ -0,0 +1,31 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+$ var confirmId = component.getElId("alert-dialog-confirm");
+$ var mainId = component.getElId("alert-dialog-main");
+<ebay-dialog-base
+    role="alertdialog"
+    open=input.open
+    classPrefix="lightbox-dialog"
+    ignore-escape=true
+    window-type="compact"
+    main-id=mainId
+    on-open("emit", "open")
+    on-close("emit", "close")
+    buttonPosition="hidden"
+    focus=confirmId
+    class=[input.class, "lightbox-dialog--mask-fade"]
+    window-class=["lightbox-dialog__compact-window--fade"]
+    ...input>
+    <${input.renderBody}/>
+    <@footer>
+        <ebay-button
+            priority="primary"
+            aria-descrbedby=mainId
```

Comment:
Typo alert! ` aria-descrbedby` should be ` aria-describedby` :-)

Reply:
Ack! Dang autocomplete! Fixed it!